lib/matplotlib/backends/backend_qt5.py

```diff
@@ -27,7 +27,7 @@
 SPECIAL_KEYS = {QtCore.Qt.Key_Control: 'control',
                 QtCore.Qt.Key_Shift: 'shift',
                 QtCore.Qt.Key_Alt: 'alt',
-                QtCore.Qt.Key_Meta: 'super',
+                QtCore.Qt.Key_Meta: 'meta',
```

Comment:
This will need to be documented as an API change (see https://matplotlib.org/api/api_changes_3.4/README.html) as anyone who is currently looking for 'super' in the strings is going to broken by this change.

On net this seems like the right thing to do (because it is better to break a few people than to propagate the inconsistency between backends).

There are `QtCore.Qt.Key_Super_L` and `QtCore.Qt.Key_Super_R`, would it make sense to put those in as well?